deployment.yaml

```diff
@@ -0,0 +1,13 @@
+deployments:
+- name: null
+  version: null
+  tags: []
+  description: null
+  schedule: {}
+  flow_name: null
+  entrypoint: null
+  parameters: {}
+  work_pool:
+    name: kubernetes-prd-internal-tools
```

Comment:
you can drop this from your deployment file if you want to just specify via cli (or vice versa)